### ISSUE TITLE
[agw] Improve internal user tracing in AGW logs

### ISSUE THREAD

**Issue by wallyrb** [magma/magma#8076]
### Your Environment (If applicable)

- **Version: 1.5**
- **Affected Component:** Access Gateway
- **Affected Subcomponent:**mme service
- **Deployment Environment:**  Any environment

### Describe the Feature

**Is your feature request related to a known issue? Please describe.**
- Improve logging of IMSI in logs, some of the logs doesn't has the IMSI in the logs.
- Some users logs are mme and others in syslogs

**Describe the solution you'd like**
Every user log(or majority), should have attached the IMSI.

**Describe alternatives you've considered**
There is no alternative, rather than filter by using mme id and enB.  But these id change every time user attach so it makes very complicate to filter all sessions.

**Additional context**

```
001470 Tue Jun 29 14:55:23 2021 7F374DAEF700 INFO  NAS-EM tasks/nas/nas_proc.c            :0363    Received Authentication Information Answer from S6A for ue_id =0x00000078
001471 Tue Jun 29 14:55:23 2021 7F374DAEF700 ERROR NAS-EM tasks/nas/nas_proc.c            :0387    INFORMING NAS ABOUT AUTH RESP ERROR CODE
001472 Tue Jun 29 14:55:23 2021 7F374DAEF700 ERROR NAS-EM tasks/nas/nas_proc.c            :0403    Auth info Rsp failure for imsi 724028018558902, experimental_error_code 5001 
001473 Tue Jun 29 14:55:23 2021 7F374DAEF700 INFO  NAS-EM tasks/nas/emm/sap/emm_cn.c      :1136    EMMCN-SAP - Received primitive EMM_CN_AUTHENTICATION_PARAM_FAIL (402)
001474 Tue Jun 29 14:55:23 2021 7F374DAEF700 ERROR NAS-EM tasks/nas/emm/Attach.c          :1235    ATTACH - Authentication procedure failed!
001475 Tue Jun 29 14:55:23 2021 7F374DAEF700 INFO  NAS-EM tasks/nas/emm/sap/emm_fsm.c     :0238    EMM-FSM   - Received event ATTACH_REJ (6) in state EMM-DEREGISTERED
001476 Tue Jun 29 14:55:23 2021 7F374DAEF700 WARNI NAS-EM tasks/nas/emm/Attach.c          :1022    EMM-PROC  - EMM attach procedure not accepted by the network (ue_id=0x00000078, cause=8)
001477 Tue Jun 29 14:55:23 2021 7F374DAEF700 WARNI NAS-EM tasks/nas/emm/LowerLayer.c      :0472    NO Valid Security Context Available
001478 Tue Jun 29 14:55:23 2021 7F374DAEF700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :0174    EMMAS-SAP - Received primitive EMMAS_ESTABLISH_REJ (207)
001479 Tue Jun 29 14:55:23 2021 7F374DAEF700 INFO  NAS-EM tasks/nas/emm/sap/emm_as.c      :2161    EMMAS-SAP - Send AS connection establish reject
001480 Tue Jun 29 14:55:23 2021 7F374DAEF700 INFO  NAS-EM tasks/nas/emm/sap/emm_send.c    :0778    EMMAS-SAP - Send Attach Reject message (cause=8)
001
```

**Resources**
Add any links or references to other useful documents or assets.

**Desired timelines**
Share any information about timeline expectations.

**Comment by ekowtaylor** [comment]
Waiting on privacy feedback from @adarshs-magma

**Comment by ardzoht** [comment]
Some examples to add as reference on how to standardize IMSI logging: https://github.com/magma/magma/issues/6396

**Comment by ardzoht** [comment]
Another PR that adds IMSI logging to standard logs: https://github.com/magma/magma/pull/1416/files

**Comment by jingtien** [comment]
I'll let @wallyrb to confirm, but I don't think there is any UX work to address in this task, it's mostly adding the missing IMSI to the logs.  If this is the case then we can take it out of the UX backlog.
